server/index.js

```diff
@@ -256,6 +256,18 @@ app.get("/*", async (req, res) => {
     if (built) {
       document = built.doc;
       bcdData = built.bcdData;
+    } else if (
+      lookupURL.split("/")[1] &&
+      lookupURL.split("/")[1].toLowerCase() !== DEFAULT_LOCALE.toLowerCase() &&
+      !CONTENT_TRANSLATED_ROOT
+    ) {
+      // Such a common mistake. You try to view a URL that is not en-US but
+      // you forgot to set CONTENT_TRANSLATED_ROOT.
+      console.warn(
+        `URL is for locale '${
+          lookupURL.split("/")[1]
+        }' but CONTENT_TRANSLATED_ROOT is not set. URL will 404.`
+      );
```

Comment:
This block of code is unrelated. Sorry!
But it's bitten me before and it bit me again when hacking on this PR. It's a cheap little add that can yield a super useful warning if testing a `/fr/docs/Web/...` URL in a local dev server where you forgot to set the CONTENT_TRANSLATED_ROOT. 
We could consider turning it into an error too.